src/main/java/com/juansecu/opentoonix/mail/VerificationTokenEntity.java

```diff
@@ -0,0 +1,36 @@
+package com.juansecu.opentoonix.mail;
+
+/* --- Java modules --- */
+import java.util.Date;
+
+/* --- Javax modules --- */
+import javax.persistence.*;
+
+/* --- Third-party modules --- */
+import lombok.Data;
+import org.hibernate.annotations.GenericGenerator;
+
+
+/* --- Application modules --- */
+import com.juansecu.opentoonix.user.models.entities.UserEntity;
+import com.juansecu.opentoonix.shared.generators.NumericIdGenerator;
+
+@Entity(name = "Verification_tokens")
+@Data
+public class VerificationTokenEntity {
+    private static final int EXPIRATION = 60 * 24;
```

Comment:
This attribute should not be present in this class. Instead, it should be given manually, because, it will not always be the same expiration time.